variables.tf

```
@@ -144,3 +146,24 @@ variable "source_image_reference_version" {
   default     = "latest"
 }
 
+variable "priority" {
+  description = "Specifies the priority of this virtual machine. Possible values are Regular and Spot. Defaults to Regular"
+  type        = string
+  default     = "Regular"
+}
+
+variable "eviction_policy" {
+  description = "Specifies what should happen when the Virtual Machine is evicted for price reasons when using a Spot instance. Possible values are Deallocate and Delete"
+  default     = "Deallocate"
+}
+
+variable "max_bid_price" {
+  description = "The maximum price you're willing to pay for this virtual machine, in US Dollars; which must be greater than the current spot price. If this bid price falls below the current spot price the virtual machine will be evicted using the eviction_policy"
+  default     = "-1"
+}
+
+variable "disable_password_authentication" {
+  description = "value"
+  type    = bool
+  default = true
```

```suggestion
variable "disable_password_authentication" {
  description = "Boolean that determines if password authentication be disabled on this virtual machine."
  type        = bool
  default     = true
```

Added a description to this variable